features/web-otp.yml.dist

```diff
@@ -0,0 +1,26 @@
+# Generated from: web-otp.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+status:
+  baseline: false
+  support:
+    chrome: "93"
+    chrome_android: "84"
+    edge: "93"
+compat_features:
+  # ⬇️ Same status as overall feature ⬇️
```

Comment:
@ddbeck is there a compute-baseline bug here? Without `compute_from`, I'd expect the overall status to match the last block, or sometimes no block.

Reply:
This was due to inconsistent ancestors checking, fixed with https://github.com/web-platform-dx/web-features/pull/1485. Merging main into this and refreshing the dists will fix it.

Reply:
Thank you, sorted now!